atlasdb-timelock-server/src/integTest/resources/singleTestServer.yml

```diff
@@ -1,4 +1,5 @@
-atomix:
+algorithm:
+  type: atomix
```

Comment:
I'm thinking of #1412 here; we might offer both PaxosTBS and JSimpleDB (which gives 4-5x perf, but would only be safe for systems which have stronger clock synchronization guarantees). We don't seem to have any solutions that dominate the rest in that sense (assuming Atomix worked, for instance, it would do better in HA but not as well in non-HA) so I think it's reasonable. Agree that we need to document this precisely, though.